torch/_jit_internal.py

```diff
@@ -231,6 +231,32 @@ def can_compile_class(cls):
     return all(has_code)
 
 
+def get_callable_argument_names(fn):
+    """
+    Gets names of all POSITIONAL_OR_KEYWORD arguments for callable `fn`.
+    Returns an empty list when other types of arguments are present.
+
+    This is used by `torch.jit.trace` to assign meaningful argument names to
+    traced functions and modules.
```

Comment:
Consider adding args and returns sections in this docstring.

Reply:
Done.